aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -45,6 +45,50 @@ static void sigmoid_kernel(TensorIterator& iter) {
   });
 }
 
+static void abs_kernel(TensorIterator& iter) {
+  AT_DISPATCH_ALL_TYPES(iter.dtype(), "abs_cpu", [&]() {
+    unary_kernel_vec(
+        iter,
+        [=](scalar_t a) -> scalar_t { return std::abs(a); },
+        [=](Vec256<scalar_t> a) {
+          return a.abs();
+        });
+  });
+}
+
+static void frac_kernel(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES(iter.dtype(), "frac_cpu", [&]() {
+    unary_kernel_vec(
+        iter,
+        [=](scalar_t a) -> scalar_t { return a - std::trunc(a); },
+        [=](Vec256<scalar_t> a) {
+          return a - a.trunc();
```

Comment:
It's better to add this to vec256 as a ".frac()' function in case we'll find a better way of doing this down the road. Same for neg below. Then you can also use the macros to get rid of the boilerplate.